### PR TITLE
[PBW-4136] upNext responsive

### PR DESCRIPTION
**Pull request by ksunika**

### CODE REVIEW

File: js/components/countDownClock.js [hunk 1]
```diff
@@ -101,6 +109,19 @@ var CountDownClock = React.createClass({
     this.context.fill();
   },
 
+  updateClockSize: function(){
+    if (this.props.controller.state.screenToShow === CONSTANTS.SCREEN.DISCOVERY_SCREEN){
+      var clockWidth = 75;
+    }
+    else {
+      var clockWidth = this.props.responsiveView == 'small' ? 25 : 36;
+    }
```
Comment (mistermendez) [inline js/components/countDownClock.js]: use `this.props.skinConfig.responsive.breakpoints.sm.name` instead of 'small'

**Comment by ksunika** [comment]
@mistermendez replaced 'smal'l by this.props.skinConfig.responsive.breakpoints.sm.name

**Comment by mistermendez** [comment]
use `componentWillReceiveProps` instead of ResizeMixin. Check for `nextProps.responsiveView` and `this.props.responsiveView`

**Comment by ksunika** [comment]
@mistermendez, look like we don't even need the componentWillReceiveProps here, works fine after I removed resizing.

**Comment by mistermendez** [comment]
:sunglasses:

**Pull request merged**